docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,103 @@
+# Tizen 6.0
+
+The second milestone (M2) release of Tizen 6.0 was announced in October 2020.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+The key features of Tizen 6.0 are the enhancement of the development environment, core, and AI components.
+
+## Enhancement of development environment and core component
+
+### Tizen.NET
+
+Tizen supports C# development environment based on .NET Core for application development productivity.
+
+In Tizen 6.0, the latest  .NETCore 3.1 and the latest version of cross-platform Xamarin are supported. With this update, there is a considerable advancement in the application start-up time and reduction in memory usage. The **XAML Hot Reload** beta service is supported as well.
+
+The following is updated in Tizen 6.0:
+
+- **.NET Core**:
+     - Latest .NET Core 3.1(LTS)
+     - Optimized performance
+     - Diagnostics tools
+- **TizenFX**:
+    - Tizen platform-specific API set
+    - API level 8 released
+- **Xamarin**:
+    -  Latest Xamarin.Forms (4.8.0)  and Xamarin.Essentials (1.5.3) are available.
+    - TV UI Controls and CircularUI: New UI controls for Smart TVs and Smart Watches.
+    - XAML Hot Reload: Boosting developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device various offloading services
```

Comment:
Is this correct? 

- Web-based device various offloading services

I think it should be: 
- Web-based device offloading services
- 
However I may be wrong, please verify.

Reply:
"Web-based device offloading services" seems to be OK. I will update it